global/hooks/notify.sh

```
@@ -18,55 +65,60 @@ bring_to_front() {
+    return 1
+}
+
+# 方法2: 親プロセス名から検出（ワイルドカード対応）
+detect_from_parent_process() {
     local ppid_check=$$
     for _ in {1..10}; do
         ppid_check=$(ps -o ppid= -p "$ppid_check" 2>/dev/null | tr -d ' ')
         [[ -z "$ppid_check" || "$ppid_check" == "1" ]] && break
-        
+
         local pname=$(ps -o comm= -p "$ppid_check" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)